tests/room_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+from ..main.app import Amity
+import sqlite3
+
+import os
+
+
+
+class roomstest(unittest.TestCase):
+    def setUp(self):
+        self.amity = Amity()
+        self.conn = sqlite3.connect("test_amity.sqlite")
+        self.connect = self.conn.cursor()
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Rooms(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, type TEXT)")
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Persons(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, Personnel_type TEXT, want_accommodation TEXT, office_accommodation TEXT , living_accomodation TEXT)")
+        self.rooms = {
+            'O': [],
+            'L': []
+        }
+
+        self.people_data = {'Staff': [],
+                            'Fellow': []
+                            }
+
+
+    def test_add_person(self):
+        self.amity.add_person('Margie', 'Rain', 'Fellow', 'Y')
+        self.name = 'Margie' + " " + 'Rain'
+        self.person_type = 'Fellow'
+        self.want_accommodation = 'Y'
+        self.assertEqual(self.name, 'Margie Rain')
```

Comment:
**Code Quality: Critical**

Why are you testing a value that you just set 3 lines above? Need to work on having a better understanding of Testing.